src/classes/VOL_CTRL_PersonalSiteContactLookup_TEST.cls

```diff
@@ -84,19 +84,23 @@ private with sharing class VOL_CTRL_PersonalSiteContactLookup_TEST {
     * exists.  if not, will run under the current user.
     * @return void
     ********************************************************************************************************/
-    private static void testPageWithSitesGuestUser() {
-        list<User> listU = [Select Username, UserType, Name, IsActive, Id From User where IsActive = true and UserType = 'Guest' and
+    private static testmethod void testPageWithSitesGuestUser() {
```

Comment:
This method is both annotated IsTest (above the comments) and typed as a testMethod - causing the build to fail.